modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataLayoutResourceTest.java

```diff
@@ -365,6 +377,10 @@ public void testPostDataLayoutContext() throws Exception {
 
 		String content = httpResponse.getContent();
 
+		JSONObject jsonObject = JSONFactoryUtil.createJSONObject(content);
+
+		Assert.assertEquals("en", jsonObject.getString("editingLanguageId"));
+
```

Comment:
can we move this code to after the Assert's of the content variable and use a variable instead of "en" ?

```
String content = httpResponse.getContent();

Assert.assertThat(content, CoreMatchers.containsString("myNamespace"));
Assert.assertThat(
	content, CoreMatchers.containsString("testContainer"));
Assert.assertThat(content, CoreMatchers.containsString("valor"));
Assert.assertThat(content, CoreMatchers.containsString("value"));

JSONObject jsonObject = JSONFactoryUtil.createJSONObject(content);

Assert.assertEquals(LocaleUtil.ENGLISH.getLanguage(), jsonObject.getString("editingLanguageId"));

Assert.assertEquals(200, httpResponse.getStatusCode());
```